Package.swift

```diff
@@ -5,7 +5,7 @@ import PackageDescription
 
 let package = Package(
     name: "hummingbird",
-    platforms: [.macOS(.v10_15), .iOS(.v13), .tvOS(.v13)],
+    platforms: [.macOS(.v13), .iOS(.v16), .tvOS(.v16)],
```

Comment:
Any particular reason for the bump?

Reply:
It's needed for protocols with primary types ie HBResponder.